.template/addons/github/.github/workflows/README.md.tt

```diff
@@ -17,3 +17,43 @@ Depending on the workflow used in the project, set up the following secrets:
 - DOCKER_IMAGE (Default: ${{ github. repository }})
 - HEROKU_API_KEY (for deployment)
 - DANGER_GITHUB_API_TOKEN (Default: ${{ github.token }})
+
+### Heroku Deployment
+
+Use the `deploy_heroku.yml` GitHub Action to deploy your app in Heroku.
+
+
+- If needed, create an account on [Heroku.com](https://www.heroku.com/)
+- Add your Heroku API Key (`HEROKU_API_KEY`) in GitHub Secrets
+- Create 2 new Heroku apps:
+    - {your-project-name}
+    - {your-project-name}-staging
+- If needed, update the `.github\workflow\deploy_heroku.yml` file to matches your Heroku apps names:
+```yml
+[...]
+      - name: Set env HEROKU_APP
+        run: |
+          if [[ $BRANCH_TAG = "latest" ]]
+          then
+            echo "HEROKU_APP=HERE" >> $GITHUB_ENV
+          else
+            echo "HEROKU_APP=HERE-staging" >> $GITHUB_ENV
+          fi
+[...]
+```
```

Comment:
since the code block belongs to the bullet point, I would suggest indenting it at the same level so it would be rendered correctly as:
<img width="578" alt="Screen Shot 2021-06-09 at 17 42 37" src="https://user-images.githubusercontent.com/1896814/121340698-16aafe80-c94a-11eb-936c-e5243ac57818.png">

instead of:
<img width="563" alt="Screen Shot 2021-06-09 at 17 41 37" src="https://user-images.githubusercontent.com/1896814/121340579-fbd88a00-c949-11eb-9226-8ad9002035a1.png">  

```suggestion
- If needed, update the `.github\workflow\deploy_heroku.yml` file to matches your Heroku apps names:
  ```yml
  [...]
        - name: Set env HEROKU_APP
          run: |
            if [[ $BRANCH_TAG = "latest" ]]
            then
              echo "HEROKU_APP=HERE" >> $GITHUB_ENV
            else
              echo "HEROKU_APP=HERE-staging" >> $GITHUB_ENV
            fi
  [...]
  ```
```